test_files/export_multi/export_multi.js

```diff
@@ -0,0 +1,27 @@
+goog.module('test_files.export_multi.export_multi');
+var module = module || { id: 'test_files/export_multi/export_multi.ts' };
+module = module;
+exports = {};
+var _a;
```

Comment:
it'd odd how for `_a` gets hoisted up here, but I presume that's just what TS does?

Reply:
I also found that pretty strange.  I will dig into this more.  It's possible this transformation nukes some important info.

(On the other hand, it passes global TAP...)

Reply:
This is unrelated to this change.  TS somehow hoists the 'var' above the fileoverview in general, so I am not too concerned about it.